coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/contributor/adapter/MergeContributorsAdapter.java

```diff
@@ -34,6 +37,7 @@ public void mergeContributors(List<Long> contributorIds, String displayName) {
       firstEntity.getEmails().addAll(contributorEntity.getEmails());
       firstEntity.getProjects().addAll(contributorEntity.getProjects());
       contributorRepository.deleteById(contributorEntity.getId());
+      logger.info("Deleted contributor with id {}", contributorEntity.getId());
```

Comment:
Wouldn't a log about deleting a contributor be confusing if we're merging them.

Reply:
I've deleted this log.